### PR TITLE
lint: Clean lint style fixes

### PR DESCRIPTION
**Pull request by zarubaf**
Running the source code through `verible-lint` revealed a couple
of lint style problems which are fixed by this commit.

### CODE REVIEW

File: CHANGELOG.md
Comment (accuminium) [inline CHANGELOG.md:15]: I suggest you add
- deprecated `sram`
- added `stream_fork_seq`

to the Changelog.

File: Bender.yml
Comment (accuminium) [inline Bender.yml:23]: `src/stream_fork_seq.sv` needs to be added

**Review by accuminium — APPROVED** [review]
The style cleanup looks good to me, thanks for the effort!

I have two points, but I think they require no second review. You might want to put them into separate commits.

**Comment by accuminium** [comment]
Two other points:
1) Which style file did you use for linting?
2) How much effort would it be to add a linter check to GitHub CI?

File: CHANGELOG.md
Comment (zarubaf) [reply CHANGELOG.md:15]: Thanks a lot, I absolutely agree. I didn't want to introduce `stream_fork_seq`. Thanks for catching that. Absolutely agree regarding the `sram`.

**Comment by zarubaf** [comment]
1. It is the default style guide from `verible` which is the lowrisc (and hence ours). I think you can do some customization to make it fit with other style guides.
2. Absolutely easy. That was my intention to add it to the CI after this commit, I already have it working for another project and it uses GitHub actions so it can be trivially copied to other repositories (for example the `axi`).

**Pull request merged**
